src/covjson_pydantic/domain.py

```diff
@@ -17,8 +17,8 @@
 
 
 class CompactAxis(CovJsonBaseModel):
-    start: float
-    stop: float
+    start: Union[int, float]
+    stop: Union[int, float]
```

Comment:
I think Python float is the correct representation of a JSON number, so we don't need int here.